test/integration/ios.yaml

```diff
@@ -112,6 +112,16 @@
 #            failed_modules: "{{ failed_modules }} + [ 'ios_vlan' ]"
 #            test_failed: true
 
+#    - block:
```

Comment:
Why is this block commented?

Reply:
We do not have l2 switch ios image for dci test.